at_onboarding_cli/lib/src/at_onboarding_service_impl.dart

```diff
@@ -319,6 +321,36 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
     }
   }
 
+  ///Method to check if secondary belonging to [_atSign] exists
+  ///If not, wait until secondary is created
+  Future<void> _waitUntilSecondaryExists() async{
```

Comment:
please dartfmt the code

Reply:
dart format changes pushed in latest commit